src/Microsoft.ML.Transforms/Text/TextCatalog.cs

```diff
@@ -157,29 +159,18 @@ public static class TextCatalog
         /// <param name="outputColumnName">Name of the column resulting from the transformation of <paramref name="inputColumnName"/>.</param>
         /// <param name="inputColumnName">Name of the column to transform. If set to <see langword="null"/>, the value of the <paramref name="outputColumnName"/> will be used as source.</param>
         /// <param name="separators">The separators to use (uses space character by default).</param>
-        public static WordTokenizingEstimator TokenizeWords(this TransformsCatalog.TextTransforms catalog,
+        public static WordTokenizingEstimator ProduceWordTokens(this TransformsCatalog.TextTransforms catalog,
```

Comment:
>ProduceWordTokens [](start = 46, length = 17)

I would suggest to keep the `Tokenize` in the name of these extensions because it is the most intuitive description the behavior of the underlying transforms.
I agree with @ivmatan@microsoft.com  that would be better to explain the difference in behavior between the methods  word level and the character level tokenizers.

How about:
`TokenizeWords`
`TokenizeCharactersAsKeys`

I believe the usual behavior of word tokenizers is that they produce text tokens, which is why I would keep the name unchanged for the word level tokenizer. 
For the character level one, we can indicate the difference with the general behavior by adding `AsKeys` at the end.

Reply:
I will do
```
TokenizeIntoWords
TokenizeIntoCharactersAsKeys
```

---
In reply to: [265253207](https://github.com/dotnet/machinelearning/pull/2916#discussion_r265253207) [](ancestors = 265253207)